docs/data_pipeline_procedure.md

```diff
@@ -12,6 +12,45 @@
       find /global/cfs/cdirs/kbase/collections/$working_dir -type d -exec chmod g+s {} \;
       ```
 
+## Setting up Python environment using Pipenv
+   * Verify Python 3.11 installation
+     * To confirm Python 3.11 installation, run:
+       ```commandline
```

Comment:
is 3.11 now default on perlmutter? I'd check but it's down

Reply:
yea. perlmutter is down now. But it's not critical; once python --version confirms Python 3.11, we can proceed with the next steps using pipenv. How one chooses to manage their base environment is entirely up to them using default or conda.

Reply:
What I mean is if a user does python --version on perlmutter will they get 3.11? I seem to remember it being pretty out of date

Reply:
perlmutter's up now. I am also surprised that the default installed python is 2.7. 

tgu@login38 collections  $ python --version
Python 2.7.18

Reply:
ew. So they probably need some help in figuring out how to get the right version running

Reply:
yea. the conda example will ensure python 3.11 is installed. But I don't think we should instruct users how to configure their python environment. That's why I linked the NERSC document regrading how users can manage their Python environment as well as the conda example.

Reply:
I made a small text suggestion that made what you're saying here clearer to me in the new docs

Reply:
👍

Reply:
`module load python` will get 3.11.6 by default on perlmutter, at least on my machine. Not sure this is useful now.

Reply:
👍 add more doc for using the python module.

Reply:
:+1: much better